### PR TITLE
Add localisation

### LINKED ISSUE #14: Localised storefronts
Localisation slows development down considerably but it's one of those things where once it's in, it's not so bad to maintain.

If we're doing it we should commit to doing it from day one of proper development, as it's always more effort to go back in after.

### PR DESCRIPTION
**Pull request by GooBall**
## Review / Testing
To check the translations are working, I have added the `pirate` locale that prepends "[ARRRR!]" to the start of every string. 🏴‍☠️ 
To see this in action, change the like 6 of `config/initializers/locale.rb` to be `I18n.default_locale = :pirate` or change your url to include `pirate/` just after the main domain (`http://127.0.0.1:3000/pirate/keys`).

Resolves #14

### CODE REVIEW

**Review by elliotcm — APPROVED** [review]

File: features/step_definitions/admin/access_steps.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@
   go_to_admin_homepage
   expect(page).not_to have_text("Donations")
   visit admin_donations_path
-  expect(page).to have_current_path("/admin")
+  expect(page).to have_current_path("/admin?locale=en")
```
Comment (GooBall) [inline features/step_definitions/admin/access_steps.rb:51]: @elliotcm This feels bad, but I've not been able to find a workaround. The fact that other checks, like the one on line 59 (`expect(page).to have_current_path(admin_donations_path)`), pass just fine makes me think that if we were to use a named route here there wouldn't be the requirement for `?locale=en`.

Thoughts?

Comment (elliotcm) [reply features/step_definitions/admin/access_steps.rb:51]: Gotcha.  You could try `admin_root_path`.

Comment (GooBall) [reply features/step_definitions/admin/access_steps.rb:51]: Ah good shout. Let me try that now

Comment (GooBall) [reply features/step_definitions/admin/access_steps.rb]: Works like a charm!

**Review by elliotcm — APPROVED** [review]
We can remove the pirate one when the first real alternate locale goes in.

**Pull request merged**